test/FlapperUniV2.t.sol

```diff
@@ -132,6 +136,29 @@ contract FlapperUniV2Test is DssTest {
         end           = EndLike(ChainlogLike(LOG).getAddress("MCD_END"));
         spotter       = SpotterLike(ChainlogLike(LOG).getAddress("MCD_SPOT"));
 
+        address farm = address(new StakingRewardsMock(PAUSE_PROXY, address(0), DAI, address(new GemMock(1_000_000 ether))));
+        SplitterInstance memory splitterInstance = FlapperDeploy.deploySplitter({
+            deployer: address(this),
+            owner:    PAUSE_PROXY,
+            daiJoin:  DAI_JOIN,
+            farm:     farm
+        });
+        splitter = Splitter(splitterInstance.splitter);
+        SplitterConfig memory splitterCfg = SplitterConfig({
+            hump:            50_000_000 * RAD,
+            bump:            5707 * RAD,
+            hop:             30 minutes,
+            burn:            WAD,
+            rewardsDuration: 30 minutes,
+            farm:            farm,
+            chainlogKey:     "MCD_FLAP_SPLIT"
+        });
+        DssInstance memory dss = MCD.loadFromChainlog(LOG);
+        vm.startPrank(PAUSE_PROXY);
+        FlapperInit.initSplitter(dss, splitterInstance, splitterCfg);
+        vm.stopPrank();
+        assertEq(dss.chainlog.getAddress("MCD_FLAP_SPLIT"), address(splitter));
```

Comment:
Not sure we need this here, should probably only be covered in the splitter test.

Reply:
Removed in 79a0021c